### PR TITLE
virt-xml-validate: add page

### PR DESCRIPTION
**Pull request by IvyCap**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

For  #5425

### CODE REVIEW

File: pages/linux/virt-xml-validate.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# virt-xml-validate
+
+> Validate libvirt XML files against a schema.
```
Comment (kbdharun) [inline pages/linux/virt-xml-validate.md:3]:
```suggestion
> Validate `libvirt` XML files against a schema.
```

File: pages/linux/virt-xml-validate.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+> If a schema is not specified, the schema is determined by the root element in the XML file.
+> More information: <https://libvirt.org/manpages/virt-xml-validate.html>.
+
+- Validate a XMl files against a specific schema:
```
Comment (kbdharun) [inline pages/linux/virt-xml-validate.md:7]:
```suggestion
- Validate an XML file against a specific schema:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**